src/validationplots/plots.jl

```diff
@@ -1,13 +1,32 @@
+"""
+# TODO : desc
+"""
+function probplot(fm::fittedEVA)::Plot
```

Comment:
This is more of a general comment. Now that we have so many functions, I believe we should rethink our structure. The one I initially thought of is not suitable anymore. 

Maybe we could split it this way:
plots.jl (contains functions that draw the plots)
plots_data.jl (contains non-standardized data functions)
plots_std_data.jl (contains standardized data functions)

We could also think of separating the functions based on the graphs (group together everything related to probability plots, qq plots, etc.)

Reply:
Done